apps/app/.env.test

```diff
@@ -1,5 +1,6 @@
 NEXT_PUBLIC_NAME=STAGING
-NEXT_PUBLIC_API_URL=https://localhost:1337/
-NEXT_PUBLIC_STRAPI_URL=https://localhost:1337/
-NEXT_PUBLIC_GOOGLE_AUTH_URL=https://localhost:1337/connect/google
+NEXT_PUBLIC_API_BASE_URL=https://api-staging.devlaunchers.org
+NEXT_PUBLIC_API_URL=https://api-staging.devlaunchers.org
```

Comment:
Do we need `https://api-staging.devlaunchers.org/api`?

Reply:
looks like this was also corrected